scripts/install_deps.sh

```diff
@@ -37,20 +37,27 @@ if [[ "${MACHINE}" == "Linux" ]]; then
         sudo apt install -y g++ gcc make ccache python3-dev gfortran
         if [ "$release_num" == "20.04" ];then
             sudo apt install -y python3-setuptools swig
+            sudo apt install libmkl-full-dev
         fi
         # Pre-installation of openblas can save about 15 minutes of openblas building time.
         # But the apt-installed openblas version is 0.2.20, while the latest openblas version is 0.3.19.
         # So we only pre-install openblas in Unittest, and compile openblas-0.3.19 when release.
         if [[ "${INSTALL_OPENBLAS}" == "true" ]]; then
           sudo apt install -y libopenblas-dev
         fi
+        #DiskANN dependencies
+        sudo apt-get install -y libboost-program-options-dev
```

Comment:
centos cannot use apt-get, please use package manager install intel mkl library.

Reply:
We are using shell script from intel to install mkl not apt in centos, which leads to an already installed error. Change to yum method.